.emacs.d/lisp/package/packup.el

```diff
@@ -10,7 +10,7 @@
 
 (package-initialize)
 
-(defvar kotct/packup-marker-char ?x
+(defvar kotct/packup-marker-char ?*
```

Comment:
I kept seeing `x` as the mark, so my brain would go to mark something and I'd hit `x` on my keyboard. Then it would execute the transaction. Changing to `*` avoids this issue and also brings the marks more in line with the rest of emacs (think dired, list-buffers, etc).